CHANGELOG.md

```
@@ -16,8 +16,9 @@ Format follows [Keep a Changelog](https://keepachangelog.com/).
 - pytest-socket for network isolation in unit tests
 
 ### Changed
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)